setuptools/command/install.py

```diff
@@ -63,9 +63,7 @@ def initialize_options(self):
             standards-based tools.
             """,
             see_url="https://blog.ganssle.io/articles/2021/10/setup-py-deprecated.html",
-            # TODO: Document how to bootstrap setuptools without install
-            #       (e.g. by unzipping the wheel file)
-            #       and then add a due_date to this warning.
+            due_date=(2025, 10, 31),
```

Comment:
That is not a lot of time. I maintain a build-system depending on this, and I know there are a lot of others. I don't think 6 months is enough time to change them.

The system I care about is Debian's Python Package toolchain (`dh-python`). We implemented install in it in late 2021, and many packages have been migrated across (but almost certainly only the easy ones). It's been on my todo list to look into pushing the rest over, but there hadn't been any urgency. I thought the `setup.py install` interface was too entrenched everywhere to be deprecated at short notice like this.

In practice, with our current release freeze, I can't even look at starting making large-scale changes to do anything about this, in the next 6 months.